lib/system/strmantle.nim

```diff
@@ -9,6 +9,61 @@
 
 # Compilerprocs for strings that do not depend on the string implementation.
 
+const digitsTable = "0001020304050607080910111213141516171819" &
```

Comment:
can you add a comment point to where this come from (or, even better, how to generate it)?

Reply:
```nim
var res = ""
for i in 0 .. 99:
  if i < 10:
    res.add "0" & i
  else:
    res.add $i
```

Reply:
i mean, as comment in the code

Reply:
Yeah I know, just want to show it :)

Reply:
Because I was not able to push commits then.